### ISSUE TITLE
Have you seen Taack/taack-plm-freecad ?

### ISSUE THREAD

**Issue by luzpaz** [grd/FreePDM#85]
https://github.com/Taack/taack-plm-freecad

**Comment by grd** [comment]
Yes, I saw it. I thought it was at the freecadweb site. He is clearly a talented programmer. However, the server is written in Java and that is alien to me (as a Go-lang programmer). The only problem however is (I think) that it is pretty dead.

Also, I don't want to brag, but my filesystem is pretty good right now. I took plenty of time there and it is working right now. But to be honest, I didn't look at the freecadweb site for a long time. I wanted to keep it quiet because I didn't have anything to show (because I worked very hard on the filesystem). Right now, I have a docker-compose file that works, a server that also works and a terminal client that also works (but needs a lot of polishing and new features need to be added). If you want to see me in person, I will be at the freecad event in Bruxelles (I don't know how many days but I think only one).

**Comment by luzpaz** [comment]
Thanks for your response. 

I won't be at the event but please record yourself demoing it. You can also make some feature request tickets to signal to others any help you would like a hand with? Especially UI/UX and polish.

**Comment by MisterMakerNL** [comment]
@grd can you do a presentation in Bruxelles?

**Comment by grd** [comment]
Of course I can give a presentation. The only question is about what? I don't have a GUI right now. I even don't have a TUI, the only thing that I have right now is a CMD. I can tell everything about my experiences during this exercise but I don't think that anyone is interested in that. This is why I was to shy to mention this in the freecadweb site. Next year then you can see a presentation, but unless you can give me some ideas then I don't think so this year. So only if you can give me some ideas.

**Comment by grd** [comment]
Maybe about the design choices that I made. Such as why use Go and not Python and why use this instead of that and maybe also about the filesystem that I use and the benefits of this. I can also show the YT movie about Fyne that I like. So... maybe. It all depends on how much time I need to spend on it. And I also need to mention this somewhere.

**Comment by grd** [comment]
@luzpaz About the feature request tickets? I thought about that too. For instance with the complicated version numbering system such as 0.1, A.A, 0.0.1 and you name it. I have quite a lot of documentation about it. The only problem is when I look at for instance FreeCAD they have quite a lot of very old tickets that people just don't want to pick up, either because they are lazy or because the system just went to complicated and they don't like C++. When you do things on your own you know when it is ready. Maybe I can do it.

**Comment by luzpaz** [comment]
FreeCAD is growing and with it more tickets. In my experience it's better to open a Feature Request. It signals interest and can create a point for others to organize around. You never know when the commit will be made. That's Open Source.

![Screenshot_20241230_153443](https://github.com/user-attachments/assets/89dca51e-b226-4c29-acef-fba498003a29)
